I don't always get new commands, but when I do, it's a MEME command.

PureKrome opened the pull request:
- Added new command: /meme.

![I don't always get new commands, but when I do, it's a MEME command.](http://upboat.me/tmimitw/I-don't-always-get-new-commands/-but-when-I-do,-it's-a-MEME-command..jpg)

PureKrome committed:
I don't always get new commands, but when I do, it's a MEME command.
- Added new command: /meme.

davidfowl commented on JabbR/Commands/MemeCommand.cs:
```diff
@@ -0,0 +1,34 @@
+            }
+
+            ChatRoom callingRoom = context.Repository.GetRoomByName(callerContext.RoomName);
+            string message = String.Format("http://upboat.me/{0}/{1}/{2}.jpg", args[0], args[1], args[2]);
```
Is there an https version?

DavidChristiansen replied:
Doesn't appear so Dave... :-1:

samandmoore replied:
so no @PureKrome needs to update this to use the https url :)

PureKrome commented:
I've [created a new issue](https://github.com/worldwidewat/upboat.me/issues/52) asking those people to hopefully buy an SSL cert. they are only around $20 or something...

> Should be space separated ("type top-line bottom-line")

EDIT: (my bad)

@cmaish  I'll double check it and make it that.

PureKrome committed:
Fixed meme command argument list typos.

PureKrome commented:
@cmaish well spotted, sir! I've fixed up the typos. The command was already correctly doing what u suggested .. so i just fixed the command args-text.

mharen commented:
Hi, I'm one of the guys behind upboat.me. SSL support just landed over there. Let me know if there's anything else I might be able to help with.

PureKrome commented:
![a](http://upboat.me/hj/HERESSSSSS/SSL-Support!.jpg)

![b](http://upboat.me/lyp/you-sir/are-awesome!.jpg)

PureKrome committed:
Updated (meme) Upboat.me route from http to https.

davidfowl commented on JabbR/Hubs/Chat.cs:
```diff
@@ -1056,6 +1056,11 @@ void INotificationService.ChangeWelcome(ChatUser user, ChatRoom room)
 
+        void INotificationService.GenerateMeme(ChatUser user, ChatRoom room, string message)
+        {
+            Send(message, room.Name);
```
Send?

PureKrome replied:
Yep - I didn' tknow what was better / appropriate.

davidfowl replied:
most of the other commands have specific methods

mharen commented:
@PureKrome, it was mentioned in the chat room the other day that an API for the meme list would be useful if anyone ever wanted to implement tab-completion of meme names.

I [added](https://github.com/worldwidewat/upboat.me/issues/53) that api here:
- http://upboat.me/api/list

Hopefully that's helpful :)
